src/test_servers.py

```diff
@@ -1,20 +1,57 @@
 """Tests to ensure that our server is properly echoing the client."""
-# NOTE: Server must be started manually before running Pytest.
-# -*- coding: utf-8 -*-
-import pytest
+# _*_ coding: utf-8 _*_
 
 TEST_LST = [
     ('Test msg 1', 'Test msg 1'),
     ('Test message 003', 'Test message 003'),
     ('The quick brown fox jumped over the lazy brown dog',
      'The quick brown fox jumped over the lazy brown dog'),
-    ('Über message', 'Über message')
+    (u'Über message', u'Über message')
 ]
 
 
-@pytest.mark.parametrize('client_msg, server_reply', TEST_LST)
-def test_server(client_msg, server_reply):
-    """Test that server returns client message."""
-    from client import client
+def test_response_status():
```

Comment:
This test (and the others, it seems) could benefit from being broken up. Asserts ought to be as few as absolutely necessary so that it's easier to isolate errors and pinpoint where things are going wrong. There are definitely cases where you would want multiple asserts in a test, such as when the function you're testing has side effects, but strive for a minimum number of assertions.

Here, for example, you should probably break this function out into a function that tests response_ok and a function that tests response_error.